api/passport/api.py

```diff
@@ -0,0 +1,92 @@
+import json
+
+import api_logging as logging
+import boto3
+from django.conf import settings
+from eth_utils.address import to_checksum_address
+from ninja import Schema
+from ninja_extra import NinjaExtraAPI
+from ninja_extra.exceptions import APIException
+from registry.api.utils import ApiKey, is_valid_address
+from registry.exceptions import InvalidAddressException
+
+log = logging.getLogger(__name__)
+
+api = NinjaExtraAPI(urls_namespace="passport")
+
+
+lambda_client = None
+
+
+def get_lambda_client():
+    global lambda_client
+    if lambda_client is None:
+        lambda_client = boto3.client(
+            "lambda",
+            aws_access_key_id=settings.S3_DATA_AWS_SECRET_KEY_ID,
+            aws_secret_access_key=settings.S3_DATA_AWS_SECRET_ACCESS_KEY,
+            region_name="us-west-2",
+        )
```

Comment:
This is ok for now, but after the migrations we should solve the access to the lambda client via proper role setup, instead of api keys.

@larisa17  please add this to the post-migration todo list